confdb/schema.go

```diff
@@ -41,6 +41,16 @@ type parser interface {
 	parseConstraints(map[string]json.RawMessage) error
 }
 
+type visibilityHolder interface {
```

Comment:
Why are these methods defined in a separate interface? Shouldn't they all apply to the schema types?